client/layout/logged-out.jsx

```diff
@@ -123,6 +125,8 @@ const LayoutLoggedOut = ( {
 		'is-grav-powered-client': isGravPoweredClient,
 		'is-grav-powered-login-page': isGravPoweredLoginPage,
 		'is-woocommerce-core-profiler-flow': isWooCoreProfilerFlow,
+		'is-magic-login': isMagicLogin,
+		'is-wpcom-magic-login': isMagicLogin && ! isJetpackLogin && ! isGravPoweredLoginPage,
```

Comment:
There are a lot of layers of styles applied to the login form. The magic email link, specifically, seems to load only for wpcom / jetpack / gravatar flows. See:

Adding the is-wpcom-magic-login class allows us to target the wpcom magic link page, without affecting that page when it appears in jetpack or gravatar styling, which have their own style overrides already. 

I added the is-magic-login class as well while I'm here because it seems like it could be useful going forward for applying styles across magic-link pages.